modules/k6/k6.go

```diff
@@ -0,0 +1,66 @@
+package k6
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// K6Container represents the K6 container type used in the module
+type K6Container struct {
+	testcontainers.Container
+}
+
+
+// WithTestScript mounts the given script into the ./test directory in the container
+// and passes it to k6 as the test to run.
+// The path to the script must be an absolute path
+func WithTestScript(scriptPath string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		script := filepath.Base(scriptPath)
+		target := fmt.Sprintf("/tests/%s", script)
+		mount := testcontainers.ContainerMount{
+			Source: testcontainers.GenericBindMountSource{
+			    HostPath: scriptPath,
+			},
+			Target: testcontainers.ContainerMountTarget(target),
+		}
+		req.Mounts = append(req.Mounts, mount)
+		req.Cmd = append(req.Cmd, target)
+	}
+}
+
+// WithOptions pass the given options to the k6 run command
+func WithOptions(options...string) testcontainers.CustomizeRequestOption {
```

Comment:
Wdyt if we rename this option to `WithCmdOptions`? I think it will be even more clear that the passed options will end up into the CMD of the image. Not a blocker though